sockeye/encoder.py

```diff
@@ -413,3 +470,173 @@ def get_rnn_cells(self) -> List[mx.rnn.BaseRNNCell]:
         Returns a list of RNNCells used by this encoder.
         """
         return self.forward_rnn.get_rnn_cells() + self.reverse_rnn.get_rnn_cells()
+
+
+class CharacterSequenceEncoder(Encoder):
```

Comment:
As mentioned above I would rename to ConvolutionalEncoder in the spirit of Lee et al.